### ISSUE TITLE
Cannot destrucutre property `userWebpackConfig` of 'undefined' or 'null'

### ISSUE THREAD

**Issue by bswank** [netlify/netlify-lambda#156]
When I use the **latest** version of this package with [this repo](https://github.com/maoberlehner/how-to-use-graphql-with-vue-apollo-components-and-netlify-functions), I receive the following error:

```
netlify-lambda: Starting server
/Users/bswank/Code/directoree/node_modules/netlify-lambda/lib/build.js:34
function webpackConfig(dir, {userWebpackConfig, useBabelrc}) {

TypeError: Cannot destructure property `userWebpackConfig` of 'undefined' or 'null'.
    at webpackConfig (/Users/bswank/Code/directoree/node_modules/netlify-lambda/lib/build.js:34:23)
    at Object.exports.watch (/Users/bswank/Code/directoree/node_modules/netlify-lambda/lib/build.js:154:26)
    at Command.<anonymous> (/Users/bswank/Code/directoree/node_modules/netlify-lambda/bin/cmd.js:55:11)
    at Command.listener (/Users/bswank/Code/directoree/node_modules/commander/index.js:315:8)
    at Command.emit (events.js:182:13)
    at Command.parseArgs (/Users/bswank/Code/directoree/node_modules/commander/index.js:651:12)
    at Command.parse (/Users/bswank/Code/directoree/node_modules/commander/index.js:474:21)
    at Object.<anonymous> (/Users/bswank/Code/directoree/node_modules/netlify-lambda/bin/cmd.js:99:9)
    at Module._compile (internal/modules/cjs/loader.js:688:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:699:10)
```

Sorry if I'm missing something simple!

**Comment by mikeyhogarth** [comment]
Can confirm that this is happening, don't think it's intentional - it has to do with this change;

https://github.com/netlify/netlify-lambda/commit/86332fb40148cb751dcf1a9d486290d730e50074

Which has inadvertedly made it mandatory to pass in additional webpack config - really simple fix (I'll put a PR in if I can get my kid to bed at a decent time) but in the meantime if you want to get unstuck, just temporarily do the following;

1) Create `webpack.config.temp.js` in your root folder
2) Put the following into it;
```
module.exports = {};
```
3) Start the server with this;
```
netlify-lambda serve src/lambda --config webpack.config.temp.js
```

**Comment by bswank** [comment]
Thanks @mikeyhogarth! I tried that but without the `--config` flag to actually pass the webpack config. Just thought I was on the wrong track. Appreciate it!

**Comment by sw-yx** [comment]
we need typescript :(

**Comment by sw-yx** [comment]
released fix in v1.4.11

**Comment by omonk** [comment]
@sw-yx 🤦‍♂  my bad!
